packages/govern-contract-utils/contracts/minimal-proxies/ERC1167ProxyFactory.sol

```diff
@@ -41,14 +41,15 @@ library ERC1167ProxyFactory {
         require(ok, _getRevertMsg(ret));
     }
 
-    function generateCode(address _implementation) internal pure returns (bytes memory code) {
-        code = new bytes(55);
-        
-        assembly {
-            mstore(add(code, 0x20), 0x3d602d80600a3d3981f3363d3d373d3d3d363d73000000000000000000000000)
-            mstore(add(code, 0x34), shl(0x60, _implementation))
-            mstore(add(code, 0x48), 0x5af43d82803e903d91602b57fd5bf30000000000000000000000000000000000)
-        }
+    function generateCreateData(address _implementation) internal pure returns (bytes memory) {
```

Comment:
Updated the name as it's not really generating code (that would be the bit after the "proxy code" comment). This data includes the constructor as well.

Reply:
Fair